src/util/fire.js

```diff
@@ -0,0 +1,24 @@
+import firebase from 'firebase';
```

Comment:
why not name the file firebase.js? you're not saving that much with calling it "fire"